[Analytics Hub] Add line chart with placeholder data to AnalyticsReportCard

rachelmcr opened the pull request:
Part of #8201 
⚠️ Depends on #8230 ⚠️ 

## Description

Adds the `AnalyticsLineChart` view to the `AnalyticsReportCard`:

* For now, the chart is displayed with placeholder data. This will be replaced with real data (interval data from the order data API response) in a following PR.
* The chart uses the same color as the `DeltaTag` background.

## Changes

* Adds placeholder chart data to `AnalyticsHubViewModel`.
* Adds a leading and trailing chart using `AnalyticsLineChart` view to `AnalyticsReportCard` and its view model.

## Testing

1. Launch the app.
2. Tap the "See More" button from the main dashboard
3. See the Analytics view loads with the sample charts on the Revenue and Orders cards.

## Screenshots

<img src="https://user-images.githubusercontent.com/8658164/204289129-47bb31e7-a64b-415e-89c8-50d0eb56fbcc.png" width="300px">

## Submitter Checklist

Update release notes:
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

rachelmcr committed:
Add analytics line chart to report card view

rachelmcr committed:
Stack delta tag and chart vertically when needed

rachelmcr committed:
Set max height to limit chart size in wide device views

rachelmcr committed:
Retain chart aspect ratio

rachelmcr committed:
Merge branch 'issue/8201-analytics-charts' into issue/8201-analytics-report-card-chart

Ecarrion committed:
Adjust column inner spacing

Ecarrion reviewed (approved):
Looks great! 🚀 

A couple of things to mention.

1. With the inclusion of the charts, I noticed that the spacing between the two columns wasn't right so I just adjusted it!

<img width="390" alt="Screen Shot 2022-11-28 at 5 36 58 PM" src="https://user-images.githubusercontent.com/562080/204396797-8724d804-f11b-4787-9e5c-b08232c23393.png">

2. To continue to improve accessibility it would be good to make chart leading aligned on accessible sizes, and allow the chart height to be dependent on the text scale factor, what do you think?

<img width="393" alt="Screen Shot 2022-11-28 at 5 40 21 PM" src="https://user-images.githubusercontent.com/562080/204397060-79e5f895-c4da-4748-bf49-c7c9a333d8fc.png">

rachelmcr commented:
> With the inclusion of the charts, I noticed that the spacing between the two columns wasn't right so I just adjusted it!

Thanks for catching and fixing that!

> To continue to improve accessibility it would be good to make chart leading aligned on accessible sizes, and allow the chart height to be dependent on the text scale factor, what do you think?

Sure! I debated about how much time to spend on design tweaks like that, but I agree with both ideas. I've added them as to-dos here: https://github.com/woocommerce/woocommerce-ios/issues/8201#issuecomment-1330390648
